spec/services/edit_edition_service_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+RSpec.describe EditEditionService do
+  include ActiveSupport::Testing::TimeHelpers
+
+  describe ".call" do
+    let(:edition) { build(:edition) }
+    let(:user) { build(:user) }
+
+    it "assigns attributes to an edition" do
+      revision = build(:revision)
+
+      expect { EditEditionService.call(edition, user, revision: revision) }
+        .to change { edition.revision }.to(revision)
+    end
+
+    it "does not save the edition" do
+      EditEditionService.call(edition, user, {})
+
+      expect(edition).to be_new_record
```

Comment:
I like the use of `be_new_record` here, but why is it important that the edition isn't saved?

Reply:
Sure, I've updated the commit message.